docs/src/main/asciidoc/deploying-to-kubernetes.adoc

```diff
@@ -175,6 +168,47 @@ quarkus.container-image.tag=1.0       #optional, defaults to the application ver
 
 The image that will be used in the generated manifests will be `quarkus/demo-app:1.0`
 
+=== Changing the generated deployment resource
+
+Beside generating a `Deployment` resource, you can also choose to get either a `StatefulSet`, or a `Job`, or a `CronJob` resource instead via `application.properties`:
```

Comment:
```suggestion
Besides generating a `Deployment` resource, you can also choose to generate either a `StatefulSet`, or a `Job`, or a `CronJob` resource instead via `application.properties`:
```
`beside` and `besides` both exists but given it's either one or another, I think it should be `besides`.